requirements.txt

```diff
@@ -3,8 +3,8 @@
 # please change the `requirements.in` file, and then follow
 # the instructions there to regenerate this file.
 #
-# This file is autogenerated by pip-compile with Python 3.8
-# by the following command:
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
```

Comment:
Why this change? This text is auto-generated by pip-compile:

https://github.com/jazzband/pip-tools/blob/8c418e89c389f3bcf8b6ff88e1145e327e7eef0f/piptools/writer.py#L126

Reply:
Ah, my bad. I was running into an issue when trying to push which said "python libraries do not match the libraries listed in the requirements.txt" so I had that file open and must have changed that accidentaly. I will change it back. I'll be more careful next time